integration-tests/package.json

```
@@ -3,16 +3,16 @@
   "version": "1.0.0",
   "description": "Integration tests for @anev/ts-mountebank ",
   "scripts": {
-    "test": "mocha --recursive ./lib",
+    "test": "mocha --recursive ./dist",
     "test-ci": "mocha --recursive --forbid-only ./dist",
     "build": "tsc",
     "pretty": "prettier -w ./**/*",
     "lint": "eslint . --ext .ts",
     "lint-fix": "eslint . --ext .ts --fix",
-    "mb:up": "docker run -d --name mb -p 2525:2525 -p 12345:12345 -p 12346:12346 bbyars/mountebank:2.6.0 mb start",
-    "mb:up-on-port-2524": "docker run -d --name mb -p 2524:2525 -p 12345:12345 -p 12346:12346 bbyars/mountebank:2.6.0 mb start",
+    "mb:up": "docker run -d --name mb -p 2525:2525 -p 12345:12345 -p 12346:12346 bbyars/mountebank:2.9.1 mb start",
+    "mb:up-on-port-2524": "docker run -d --name mb -p 2524:2525 -p 12345:12345 -p 12346:12346 bbyars/mountebank:2.9.1 mb start",
     "test-different-mb-url": "MB_PORT=2524 npm run test-ci -- -g ^Mountebank",
-    "mb:down": "docker kill mb && docker rm mb"
+    "mb:down": "docker kill mb && podman rm mb"
```

```suggestion
    "mb:down": "docker kill mb && docker rm mb"
```

Did this get changed accidentally?

---

Fixed. Sorry for the noob mistake.